examples/gcoap/client.c

```diff
@@ -141,18 +141,46 @@ static void _resp_handler(const gcoap_request_memo_t *memo, coap_pkt_t* pdu,
     }
 }
 
-static bool _parse_endpoint(sock_udp_ep_t *remote,
-                            const char *addr_str, const char *port_str)
+static bool _parse_endpoint(sock_udp_ep_t *remote, const char *addr_str, const char *port_str)
 {
-    netif_t *netif;
+    bool is_ipv4 = false;
+    bool is_ipv6 = false;
 
-    /* parse hostname */
-    if (netutils_get_ipv6((ipv6_addr_t *)&remote->addr, &netif, addr_str) < 0) {
+    if (IS_ACTIVE(SOCK_HAS_IPV6)) {
+        netif_t *netif;
+        if (strstr(addr_str, ":") != NULL) { /* IPv6 might contain '.', but IPv4 cannot contain ':' */
```

Comment:
```suggestion
        /* IPv6 might contain '.', but IPv4 cannot contain ':' */
        if (strstr(addr_str, ":") != NULL) {
```

To please the CI, could you split the line to not exceed 100 chars?